lib/google_sign_in_desktop.dart

```diff
@@ -456,6 +460,91 @@ class GoogleSignInDesktop extends GoogleSignInPlatform {
       await server.close();
     }
   }
+
+  @override
+  // ignore: avoid_renaming_method_parameters
+  Future<bool> requestScopes(List<String> requestedScopes) async {
+    if (requestedScopes.isEmpty) {
+      return false;
+    }
```

Comment:
Could we throw an `ArgumentError` if the list of `scopes` is empty?

```suggestion
    if (requestedScopes.isEmpty) {
      throw ArgumentError.value(
        scopes,
        'scopes',
        'Scopes must not be empty',
      );
    }
```